doc/rst/usingchapel/tasks.rst

```diff
@@ -357,34 +357,30 @@ CHPL_TASKS == qthreads
   environment variable.  See the qthreads subsection of `Task
   Implementation Layers`_ for more information.
 
-----------------------------------------------
-Task-Related Quantification Methods on Locales
-----------------------------------------------
+----------------------------
+Quantifying Tasks on Locales
+----------------------------
 
-Several methods on the locale type are available to query the state of
-the program with respect to its tasks.
+The locale type has a method available to query the number of tasks that
+are running on a given locale.
+
+  runningTasks()
+    returns the number of tasks that have been created
+    but have not yet finished.  Note that this number can exceed the
+    number of threads because tasking layers may be capable of switching
+    among multiple Chapel tasks running on a single hosting thread.
+
+In order to use this method, you have to specify the locale you wish to
+query, as in here.runningTasks(), where 'here' is the current locale.
+
+The following methods are also available, but these are all deprecated
+and may be expected to be removed in the next release unless a need for
```

Comment:
"may be expected to be removed" reads a little odd. Maybe just `are expected to be removed"

Reply:
Will do.